src/into_dart.rs

```diff
@@ -355,3 +355,10 @@ impl<T> IntoDart for *mut T {
         }
     }
 }
+
+#[cfg(feature = "chrono")]
+impl IntoDart for chrono::DateTime<chrono::Utc> {
+    fn into_dart(self) -> DartCObject {
+        self.timestamp_micros().into_dart()
+    }
+}
```

Comment:
Looks good, but this only for native code, right? not WASM/JS from dart? do you think it will be handled here or in a different way on upstream projects who is using `allo-isolate`?

Reply:
You're right, that's what I missed at first, they are completely unrelated.